Adds download arrows to site sections that trigger a download

bitbyt3r opened the pull request:
I got annoyed that I never know which site map links will trigger a download vs which ones are just links to other pages.

This PR will add a ⤓ symbol to the link for any page that triggers a download.

![download_arrows](https://user-images.githubusercontent.com/1031406/215259153-e3574b3e-9b88-4b13-b527-8dcdc8da5059.png)

kitsuta reviewed (commented):
I love this idea!

I think we can further improve this by making the csv_file and xlsx_file decorators automatically add the download flag to the page. Also, I'm a little confused on the other changes showing up in the PR... what branch is this based off of? We should be working off `main` for future years, right?

bitbyt3r commented:
Oops, looks like I messed up the rebase originally. I cleaned up the diff and based it on main, and added the property to the existing decorators. That made a much smaller diff.

kitsuta commented on uber/site_sections/accounts.py:
```diff
@@ -88,7 +88,7 @@ def delete(self, session, id, **params):
         raise HTTPRedirect('index?message={}', 'Account deleted')
 
-    @site_mappable
+    @site_mappable(download=True)
```
This isn't a downloadable file! It lets you assign admin accounts to volunteers in bulk.

bitbyt3r replied:
Oops, should be fixed!

kitsuta commented on uber/site_sections/guest_reports.py:
```diff
@@ -96,7 +96,7 @@ def rock_island(self, session, message='', only_empty=None, id=None, **params):
         }
 
-    @site_mappable
+    @site_mappable(download=True)
```
This isn't a problem, just curious -- is this because the function has a site_mappable tag (which sets download to False) in addition to the CSV file tag (which sets download to True)?

bitbyt3r replied:
Yeah, pretty sure that's what was happening. I could fix it by flipping the order of the decorators, but I think that'd be a bit obtuse so I just made it explicit.

kitsuta reviewed (changes requested):
One of the pages you marked as downloadable is a regular page -- other than that, this looks great!
